README.md

```diff
@@ -11,7 +11,47 @@ Before running the `run.sh` script, open the file `secrets.sh.example` and renam
 
 The `run.sh` script will use the environment variables listed in `secrets.sh` and launch three docker containers. The API server will be available at http://localhost:1235.
 
-In order to access any of the REST API endpoints, you will need to get an access code from CILogon and exchange it for an access token:
+## Overview
+The authorization service allows PCGL to separately manage OIDC tokens and clients for different services while still using CILogon/COManage to handle authentication and user roles and groups.
+
+The authorization service manages policy decisions about user access for registered services as well.
+
+PCGL services will register with the authorization service. Registration allows authz to:
+* designate actions allowed for users and user groups, based on study
+* designate an OIDC client that will be the issuer of its user tokens
+* allows the authz service to verify that any authz call is coming from a known registered service.
+
+## Registering a service
```

Comment:
This information should be moved to (and harmonized with) the info in https://github.com/Pan-Canadian-Genome-Library/pcgl-authz/blob/main/docs/service-registration.md